src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -127,7 +135,9 @@ namespace {lc.Namespace}
             private void GenStruct(LoggerMethod lm, string nestedIndentation)
             {
                 _builder.AppendLine($@"
+        {nestedIndentation}/// {s_generatedTypeSummary}
```

Comment:
I don't this this would work if `s_generatedTypeSummary` has multiple lines, does it?

Reply:
nvm, seems like the summary doesn't change.